### ISSUE TITLE
Add default export and include import line in README

### ISSUE THREAD

**Issue by exogen** [electrode-io/above-the-fold-only-server-render#4]
There's currently nothing in the README showing how to import the `AboveTheFoldOnlyServerRender` component.

A reasonable assumption would be:

```jsx
import AboveTheFoldOnlyServerRender from "above-the-fold-only-server-render";
```

Which will not error out but rather give a confusing warning message from React when you render it (you're effectively rendering `null`).

The actual import should be:

```jsx
import { AboveTheFoldOnlyServerRender } from "above-the-fold-only-server-render";
```

but there's nothing in the README indicating this. (IMO, both ways should work – it can be both a named and `default` export.)

**Comment by caoyangs** [comment]
https://github.com/electrode-io/above-the-fold-only-server-render/pull/3
Added `import { AboveTheFoldOnlyServerRender } from "above-the-fold-only-server-render";`

**Comment by markbrouch** [comment]
@exogen Documentation should also be added for the `placeholder` and related props.

**Comment by caoyangs** [comment]
`+ above-the-fold-only-server-render@1.0.3`

**Comment by ananavati** [comment]
@exogen can we please close out this issue if the fix is published in `+ above-the-fold-only-server-render@1.0.3`

**Comment by exogen** [comment]
@ananavati Yup; should've just been closed when #3 was merged.
